src/ppx/graphics_util.cpp

```diff
@@ -741,7 +748,7 @@ Result CreateImageFromFile(
             ppxres = CreateImageFromBitmapGpu(pQueue, &bitmap, ppImage, options);
```

Comment:
return error if imageDepth != 1 since it's not implemented yet for the Gpu version

Reply:
alternatively as we discussed, you can skip changing this function `CreateImageFromFile` so you don't need to worry about the GPU version. Only change `CreateImageFromBitmap` and use that function in our project to create  3D image.